packages/elvis/src/components/button/tertiary.scss

```diff
@@ -36,7 +36,7 @@
   @include buttonHelper.hover() {
     border-color: var(--e-color-border-hover-1);
     background: none;
-    color: var(--e-color-text-1);
+    color: inherit;
```

Comment:
Trenger vi å si `inherit` her og på linje 50?

Reply:
Det er ikke sikkert vi trenger de, men etter å ha kjørt tester uten å inkludere color: var() eller color: inherit vil ikke ikon og tekst bli synlig når knappene er trykket på eller blir hoveret over. De defaulter til svart tekst, så da får vi det samme problemet som med alert hvor den endrer tekst og ikon basert på tema, og vi får svart tekst på svart bakgrunn.

Reply:
Det ser for meg ut som den da faller tilbake på å bruke `--e-color-background-hover-1--contrast) , som bør være riktig 🤔  Litt merkelig at testene feiler

![image](https://github.com/user-attachments/assets/a8050b20-2b6e-4ca5-a1aa-0ddc96e27112)

Reply:
Når det gjelder * {color: var(--e-color-text-1)}; så fungerer det på alert ser det ut som, men den på virker også alle andre elementer på siden hvor tertiary.scss blir brukt, så da får vi f.eks. problemer på knapper inne i alert komponenten som ikke tilpasser seg riktig.